app/src/main/java/com/github/kulebin/myfoursquareapp/api/Api.java

```diff
@@ -0,0 +1,48 @@
+package com.github.kulebin.myfoursquareapp.api;
+
+import android.net.Uri;
+
+import com.github.kulebin.myfoursquareapp.BuildConfig;
+
+public final class Api {
+
+    private static final String BASE_URL = "https://api.foursquare.com/v2";
+
+    private static final String PATH_VENUES = "venues";
+    private static final String PATH_TRENDING = "trending";
+
+    private static final String PARAM_DATE_VERIFIED = "v";
+    private static final String PARAM_LATITUDE_AND_LONGITUDE = "ll";
+    private static final String PARAM_LIMIT = "limit";
+    private static final String PARAM_RADIUS = "radius";
+    private static final String PARAM_CLIENT_ID = "client_id";
+    private static final String PARAM_CLIENT_SECRET = "client_secret";
+
+    //temporarily hardcoded values
+    private static final String VALUE_DATE_VERIFIED = "20170226"; /*Date value format: YYYYMMDD */
+    private static final String VALUE_LATITUDE_AND_LONGITUDE_MINSK = "53.9,27.56667";
+
+    private static Uri.Builder getVenuesUri() {
```

Comment:
As this method uses only static Strings, I want to see Uri baseUrl as a static field inited once and used later to create urls.